std_msgs/String     type_parameter
geometry_msgs/Twist trajectory_characteristic